### PR TITLE
feat(schema): Embed column creation options

### PR DESCRIPTION
**Pull request by candiduslynx**

### CODE REVIEW

File: schema/meta.go [hunk 1]
```diff
@@ -14,45 +14,48 @@ type ClientMeta interface {
...
 }
 var CqSourceNameColumn = Column{
-	Name:        "_cq_source_name",
-	Type:        arrow.BinaryTypes.String,
+	Name:        CqSourceNameField.Name,
+	Type:        CqSourceNameField.Type,
 	Description: "Internal CQ row that references the source plugin name data was retrieved",
 }
 
 var CqIDField = arrow.Field{
```
Comment (yevgenypats) [inline schema/meta.go:42]: I think this can be removed

File: schema/meta.go [hunk 2]
```diff
@@ -14,45 +14,48 @@ type ClientMeta interface {
...
-	Type: types.ExtensionTypes.UUID,
-	Metadata: arrow.MetadataFrom(map[string]string{
-		MetadataUnique: MetadataTrue,
-	}),
+	Name:     "_cq_id",
+	Type:     types.ExtensionTypes.UUID,
+	Metadata: arrow.MetadataFrom(map[string]string{MetadataUnique: MetadataTrue}),
 }
+
+var CqParentIDField = arrow.Field{
```
Comment (yevgenypats) [inline schema/meta.go:48]: I think this can be deleted as well

**Review by yevgenypats — CHANGES_REQUESTED** [review]
Looks good. small comment.

File: schema/meta.go (hunk 1 above)
Comment (candiduslynx) [reply schema/meta.go]: done

**Review by yevgenypats — CHANGES_REQUESTED** [review]
Why rename `ColumnCreationOptions ` to `CreationOptions` if we are doing breaking change we can remove the struct all together

**Comment by candiduslynx** [comment]
> Why rename `ColumnCreationOptions ` to `CreationOptions` if we are doing breaking change we can remove the struct all together

I think the options should be grouped together, so having them in a separate struct might be better.
The impact of removing can be seen in 1615ad9fe0b0f39f75003899aa3c4f2b8e040712

**Review by yevgenypats — APPROVED** [review]

**Review by yevgenypats — APPROVED** [review]

**Pull request merged**